src/sage/libs/gap/gap_test.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+from sage.libs.gap.libgap import libgap
+
+
+@pytest.fixture
+def tmpfile():
```

Comment:
is there nothing existing like this in pytest/sage? This feels oddly general.

Reply:
Pytest has a built-in `tmpdir` that does almost the same thing, but with a directory instead of a file. But considering https://github.com/pytest-dev/pytest/issues/13669, this is safer for now.